openc3/spec/models/target_model_spec.rb

```diff
@@ -332,6 +332,26 @@ module OpenC3
         tf.puts "CMD_LOG_CYCLE_SIZE 2"
         tf.puts "CMD_DECOM_LOG_CYCLE_TIME 3"
         tf.puts "CMD_DECOM_LOG_CYCLE_SIZE 4"
+        tf.puts "CMD_BUFFER_DEPTH 9"
+        tf.puts "CMD_LOG_RETAIN_TIME 10"
+        #tf.puts "CMD_LOG_RETAIN_SIZE 11"
+        tf.puts "CMD_DECOM_LOG_RETAIN_TIME 12"
+        tf.puts "TLM_BUFFER_DEPTH 13"
+        tf.puts "TLM_LOG_RETAIN_TIME 14"
+        tf.puts "TLM_DECOM_LOG_RETAIN_TIME 15"
+        tf.puts "REDUCED_MINUTE_LOG_RETAIN_TIME 16"
+        tf.puts "REDUCED_HOUR_LOG_RETAIN_TIME 17"
+        tf.puts "REDUCED_DAY_LOG_RETAIN_TIME 18"
+        tf.puts "LOG_RETAIN_TIME 19"
+        tf.puts "REDUCED_LOG_RETAIN_TIME 20"
+        tf.puts "REDUCER_DISABLED 21"
+        tf.puts "REDUCER_MAX_CPU_UTILIZATION 22"
+        tf.puts "REDUCED_MAX_CPU_UTILIZATION 23"
+        tf.puts "CLEANUP_POLL_TIME 24"
+        tf.puts "TARGET_MICROSERVICE CLEANUP"
+        #tf.puts "PACKET REDUCER"
+        #tf.puts "PACKET DECOM"
```

Comment:
If you change the TARGET_MICROSERVICE to something other than REDUCER or CLEANUP these lines should work

Reply:
Yes, done.